serverless.yml

```diff
@@ -84,11 +84,59 @@ resources:
                 - id
                 - systemIds
 
+    CloudFrontDistribution:
+      Type: AWS::CloudFront::Distribution
+      Properties:
+        DistributionConfig:
+          Aliases:
+            - ${self:custom.aliases.${self:provider.stage}}
+          PriceClass: PriceClass_100
+          ViewerCertificate:
+            AcmCertificateArn: arn:aws:acm:us-east-1:402949050862:certificate/1ab6042c-b6df-40f9-abdc-9ab8d8df1679
+            MinimumProtocolVersion: TLSv1.2_2018
+            SslSupportMethod: sni-only
+          DefaultCacheBehavior:
```

Comment:
Does this disable all caching? Wondering if it would be possible to only diable on /index.html ?

Reply:
This app isn't a SPA and so the caching rules would need to be more complex than that. I agree with not disabling all caching at the CloudFront level though, I _think_ we should be fine to leave out `MaxTTL` and `MinTTL` then at least if we do set caching headers on static content or other things in our API they'll be respected.

Reply:
Yep, will address the caching stuff in a future PR when we have an app running :)

Reply:
👍